src/verifier.spec.ts

```diff
@@ -73,6 +73,18 @@ describe("Verifier Spec", () => {
 			});
 		});
 
+		context("when user specifies invalid monkeypatch", () => {
+			it("should return an error on invalid path", () => {
+				expect(() => {
+					verifierFactory({
+						providerBaseUrl: "http://localhost",
+						pactUrls: ["http://idontexist"],
+						monkeypatch: "/this/file/does/not/exist.rb"
+					});
+				}).to.throw(Error);
+			});
+		});
+
```

Comment:
The CLI can really only return a couple of responses. It can return an exit code 0, or an exit code 1. The only thing you can do if you get an error response is to spit out the error message, and fail the calling code. Personally, I wouldn't have this test, because all the "how I handle the exit error code" tests are going to be the same. All you need to test is "how should the client behave when it gets an error", not "what are all the different things that can cause an error". Theoretically, the underlying Ruby process could change its implementation to just ignore a missing monkeypatch file, and then this test would fail, so it's not actually giving you any feedback on the javascript code.

Reply:
This test exercises src/server.ts:125 ff. where I check if the named file exists. If not, the CLI is never called, but an Error is thrown.